examples/historical_scanning/main.rs

```diff
@@ -2,7 +2,7 @@ use alloy::{providers::ProviderBuilder, sol, sol_types::SolEvent};
 use alloy_node_bindings::Anvil;
 
 use event_scanner::{
-    EventFilter, EventScannerBuilder, Message, robust_provider::RobustProviderBuilder,
+    EventFilter, EventScannerBuilder, Message, robust_provider::builder::RobustProviderBuilder,
```

Comment:
```suggestion
    EventFilter, EventScannerBuilder, Message, robust_provider::RobustProviderBuilder,
```
Being able to import `RobustProviderBuilder` from `event_scanner::robust_provider` is fine, just not from root `event_scanner`